javaslang/src/main/java/javaslang/collection/TreeMap.java

```diff
@@ -5,11 +5,16 @@
  */
 package javaslang.collection;
 
-import javaslang.*;
+import javaslang.Kind2;
```

Comment:
The modified files had too many warnings, reduced and extracted them to simplify the review of next commit